uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/LeaveRequest.php

```diff
@@ -13,6 +13,14 @@ function _civicrm_api3_leave_request_create_spec(&$spec) {
     'type' => CRM_Utils_Type::T_BOOLEAN,
     'api.required' => 0,
   ];
+
+  $spec['toil_to_accrue'] = [
+    'name' => 'toil_to_accrue',
+    'title' => 'TOIL to accrue amount',
+    'description' => 'Amount of TOIL to accrue for the request',
+    'type' => CRM_Utils_Type::T_FLOAT,
+    'api.required' => 0,
+  ];
```

Comment:
Instead of duplicating all this information that is coming from the LeaveRequest DAO, perhaps we can just unset what we don't need. Can you please check if something would work?

```php
unset($spec['toil_to_accrue']['pseudoconstant']);
```

Reply:
This works!